ci-operator/step-registry/sandboxed-containers-operator/install-trustee-operator/sandboxed-containers-operator-install-trustee-operator-commands.sh

```
@@ -0,0 +1,702 @@
+      runAsNonRoot: true
+      seccompProfile:
+        type: RuntimeDefault
+      capabilities:
+        drop:
+        - ALL
+  restartPolicy: Never
+EOF
+
+  # Wait for pod to be ready
+  echo ">>> Waiting for kbs-client pod to be ready"
+  if ! retry oc wait --for=condition=Ready --timeout=120s pod/${kbs_client_pod} -n ${kbs_client_namespace}; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)